### PR TITLE
Issues API and ManualSelected.ImageChanged event

### PR DESCRIPTION
**Pull request by iwatkot**

### CODE REVIEW

File: supervisely/api/issues_api.py [hunk 1]
```diff
@@ -0,0 +1,481 @@
...
+
+    def add(
+        self,
+        team_id: int,
+        issue_name: str,
+        comment: Optional[str] = None,
+        assignees: Optional[List[int]] = None,
+        is_local: bool = False,
+    ) -> IssueInfo:
+        response = self._api.post(
```
Comment (GoldenAnpu) [inline supervisely/api/issues_api.py:192]: dosctring ?
`is_local` means visible only for your team?
If yes, I would propose to use `team_scope_only`

File: supervisely/api/issues_api.py [hunk 2]
```diff
@@ -0,0 +1,481 @@
...
+        return (
+            self._convert_json_info(response.json(), skip_missing=True)
+            if (response is not None)
+            else None
+        )
+
+    def add(
+        self,
+        team_id: int,
+        issue_name: str,
```
Comment (GoldenAnpu) [inline supervisely/api/issues_api.py:187]: title

File: supervisely/api/issues_api.py [hunk 3]
```diff
@@ -0,0 +1,481 @@
...
+        comment: Optional[str] = None,
```
Comment (GoldenAnpu) [inline supervisely/api/issues_api.py:188]: description

File: supervisely/api/issues_api.py [hunk 4]
```diff
@@ -0,0 +1,481 @@
...
+        # * At the moment API returns only ID of the issue (e.g. {"id": 123}).
+        # * So, we're making extra request to get full info about the issue.
+        # * Consider to update API to return full info about the issue.
+        return self.get_info_by_id(issue_id)
+
+    def update(
+        self,
+        issue_id: int,
+        issue_name: Optional[str] = None,
+        status: Optional[str] = None,
```
Comment (GoldenAnpu) [inline supervisely/api/issues_api.py:212]: Literal?

File: supervisely/annotation/label.py [hunk 5]
```diff
@@ -351,8 +356,19 @@ def from_json(cls, data: Dict, project_meta: ProjectMeta) -> LabelBase:
             description=data.get(LabelJsonFields.DESCRIPTION, ""),
             binding_key=binding_key,
             smart_tool_input=smart_tool_input,
+            sly_id=data.get(LabelJsonFields.ID),
```
[the remark this answers is not in the document]
Comment (iwatkot) [reply supervisely/annotation/label.py:359]: @GoldenAnpu the sly_id parameter can be None if working with a local project, so it's an optional parameter.

File: supervisely/imaging/image.py [hunk 6]
```diff
@@ -1359,6 +1359,22 @@ def get_labeling_tool_url(
     return res
 
 
+def get_new_labeling_tool_url(dataset_id: int, image_id: int) -> str:
```
[the remark this answers is not in the document]
Comment (iwatkot) [reply supervisely/imaging/image.py:1362]: @GoldenAnpu, there is an assumption that this method will be used in subsequent implementations where the version will be passed as an argument, so I would not like to use the version in the function name.

File: supervisely/api/issues_api.py [hunk 7]
```diff
@@ -0,0 +1,481 @@
...
+            # Load secrets and create API object from .env file (recommended)
+            # Learn more here: https://developer.supervisely.com/getting-started/basics-of-authentication
+            if sly.is_development():
+               load_dotenv(os.path.expanduser("~/supervisely.env"))
+
+            api = sly.Api.from_env()
+
+            # Get information about the issue by its ID.
+            issue_info = api.issues.get_info_by_id(1)"""
+        response = self._get_response_by_id(id, "issues.info", id_field=ApiField.ID)
```
[the remark this answers is not in the document]
Comment (iwatkot) [reply supervisely/api/issues_api.py:177]: @GoldenAnpu at the moment, this argument is not required anywhere and it is unlikely that it will be needed.

**Review by GoldenAnpu — APPROVED** [review]

**Pull request merged**